src/mds/nameserver2/curvefs.cpp

```diff
@@ -138,6 +141,7 @@ bool CurveFS::Init(std::shared_ptr<NameServerStorage> storage,
     defaultSegmentSize_ = curveFSOptions.defaultSegmentSize;
     minFileLength_ = curveFSOptions.minFileLength;
     maxFileLength_ = curveFSOptions.maxFileLength;
+    defaultPoolset_ = curveFSOptions.defaultPoolset;
```

Comment:
We do not need `defaultPoolset`. All topo related information should come from curveadm's configuration of the cluster.

Reply:
No, we need, this is for the allocation compatibility of old volumes which created in old physicalPools

Reply:
> We do not need `defaultPoolset`. All topo related information should come from curveadm's configuration of the cluster.

No, we need, this is for the allocation compatibility of old volumes which created in old physicalPools